pkg/node/runtime/cri/docker/events.go

```diff
@@ -120,17 +129,17 @@ func (r *Runtime) Subscribe(ctx context.Context, state *state.PodState, p chan s
 					log.Errorf("Container: can-not inspect")
 					break
 				}
-				if c == nil {
-					log.Errorf("Container: container not found")
-					break
-				}
 
 				state.SetContainer(container)
+
+
```

Comment:
are this four lines necessary?

Reply:
@kirillDanshin what did you mean? Which lines?

Reply:
@undassa I mean empty lines at 134-137